Controller/ContactController.php

```diff
@@ -0,0 +1,292 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContactExtensionBundle\Controller;
+
+use Hateoas\Configuration\Exclusion;
+use Hateoas\Representation\CollectionRepresentation;
+use JMS\Serializer\SerializationContext;
+use Sulu\Bundle\ContactBundle\Controller\ContactController as SuluContactController;
+use Sulu\Bundle\ContactBundle\Util\IndexComparatorInterface;
+use Sulu\Bundle\ContactExtensionBundle\Entity\Contact;
+use Sulu\Component\Rest\Exception\EntityNotFoundException;
+use Sulu\Component\Rest\Exception\MissingArgumentException;
+use Sulu\Component\Rest\Exception\RestException;
+use Sulu\Component\Rest\ListBuilder\Doctrine\DoctrineListBuilder;
+use Sulu\Component\Rest\ListBuilder\Doctrine\DoctrineListBuilderFactory;
+use Sulu\Component\Rest\ListBuilder\Doctrine\FieldDescriptor\DoctrineFieldDescriptor;
+use Sulu\Component\Rest\ListBuilder\ListRepresentation;
+use Sulu\Component\Rest\RestHelperInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+class ContactController extends SuluContactController
+{
+    /**
+     * lists all contacts
+     * optional parameter 'flat' calls listAction.
+     *
+     * @param Request $request
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function cgetAction(Request $request)
+    {
+        $serializationGroups = [];
+        $locale = $this->getLocale($request);
+
+        if ($request->get('flat') == 'true') {
+            $list = $this->getList($request, $locale);
+        } else {
+            if ($request->get('bySystem') == true) {
+                $contacts = $this->getContactsByUserSystem();
+                $serializationGroups[] = 'select';
+            } else {
+                $contacts = $this->getDoctrine()->getRepository(
+                    $this->container->getParameter('sulu.model.contact.class')
+                )->findAll();
+                $serializationGroups = array_merge(
+                    $serializationGroups,
+                    static::$contactSerializationGroups
+                );
+            }
+            // convert to api-contacts
```

Comment:
i.m.o it should be 
```
// Convert to api-contacts.
```
At least a capitalized letter at the beginning would be nice ;)